tests/e2e/test_validate_treatment.py

```diff
@@ -1,16 +1,18 @@
 from xp_client import XPClient
 
+def test_validate_valid_treatment_with_url(xp_client: XPClient, httpserver):
+    # Mock external validation endpoint
+    httpserver.expect_request("/external", method="POST").respond_with_json({})
 
-def test_validate_valid_treatment_with_url(xp_client: XPClient):
     data = {
-        "validation_url": "http://www.mocky.io/v2/5e4cacd4310000e1cad8c073",
+        "validation_url": httpserver.url_for("/external"),
```

Comment:
The old mocky URL had expired so the e2e failed. Replacing that with a local test httpserver.